samples/bootloader/README.rst

```diff
@@ -1,79 +1,63 @@
 .. _bootloader:
 
-Bootloader
-##########
+Immutable bootloader
+####################
 
-The bootloader sample implements a first stage bootloader with the capability to verify and boot the second stage bootloader.
-In cooperation with the second stage bootloader, it also provides a mechanism to upgrade the second stage bootloader.
+The bootloader sample implements an immutable first stage bootloader that has the capability to verify and boot a second stage bootloader.
+If the second stage bootloader is upgradable, this sample also provides a mechanism to upgrade the second stage bootloader.
```

Comment:
It is important that the first stage bootloader does not actually perform the update procedure. It is the second stage bootloader which updates itself, and the first stage bootloader simply chooses the one of the two candidates which has the highest version number.

Reply:
OK, updated. (I amended the commit since this is the only change.)